resources/js/components/CollapsibleResourceManager.vue

```diff
@@ -48,7 +48,7 @@
 
                 </div>
 
-                <Badge :label="data.badge">
+                <Badge :label="data.badge"  v-bind="topLevelLink">
```

Comment:
@IGedeon why binding `topLevelLink` to badge?

Reply:
this change actually breaks when you set no `linkTo` to a Group because 

1 - It nows shows with a different style (white and with a bigger front) 
2 - The cursor indicates it is clickable but because you moved the click to the a tag now it only accepts clicks on the (+/-) icon

Reply:
> @IGedeon why binding `topLevelLink` to badge?

Hi, our designer has requested us to have a collapsible menu "showing" the relationships between models. 

If we have States and Cities they want Cities to be collapsed inside States... but States has to remain clickable.

Reply:
> this change actually breaks when you set no `linkTo` to a Group because
> 
> 1 - It nows shows with a different style (white and with a bigger front) even thou its not a link
> 2 - The cursor indicates it is clickable but because you moved the click to the a tag now it only accepts clicks on the (+/-) icon

if we allow the link and the collapse/expand simultaneously how would be the best way to accomplish that?

Reply:
I mean if it's a link, it should not colappse on click, however if you don't use the linkTo perhaps it should keep the current behavior otherwise user will stay clicking and nothing will happen even thou the cursor changes into a pointer indicating you can click on it.

What I'm saying is that the changes you made only fix your use case but for those not using the LinkTo behavior will be left in a wierd state